core/store/orm/orm.go

```diff
@@ -296,6 +297,41 @@ func (orm *ORM) CreateJobRun(run *models.JobRun) error {
 	return orm.DB.Create(run).Error
 }
 
+// AddLinkEarned adds link earning
+func (orm *ORM) AddLinkEarned(earning *models.LinkEarned) error {
+	return orm.DB.Create(earning).Error
+}
+
+// LinkEarningsFor lists the individual link earnings for a job
+func (orm *ORM) LinkEarningsFor(jobSpecID string, limit ...int) ([]models.LinkEarned, error) {
+	earnings := []models.LinkEarned{}
+	var lim int
+	if len(limit) == 0 {
+		lim = 100
+	} else if len(limit) >= 1 {
+		lim = limit[0]
+	}
+	err := orm.DB.
+		Limit(lim).
+		Where("job_spec_id = ?", jobSpecID).
+		Find(&earnings).Error
+	return earnings, err
+}
+
+// LinkEarnedFor shows the total link earnings for a job
+func (orm *ORM) LinkEarnedFor(jobSpecID string) (*assets.Link, error) {
+	all, err := orm.LinkEarningsFor(jobSpecID)
+	if err != nil {
+		fmt.Println(err)
+		return nil, err
+	}
+	sum := assets.NewLink(0)
+	for _, ear := range all {
+		sum = sum.Add(sum, ear.Earned)
```

Comment:
It seems like we are avoiding summing in SQL because LINK is stored as a string in the database. As is, this implementation is problematic for the following reasons:

1. Erroneously caps the summed job specs at 100. If we made money on 101 job specs, we would have the wrong answer I believe.
2. Slower because done in memory, but that's negligible.